scripts/count_hours.py

```diff
@@ -0,0 +1,59 @@
+import warnings
+import argparse
+import librosa
+SAMPLE_RATE = 16000
+
+
+def read_audio_and_text(inputs):
+    audio_path = inputs['file_path']
+    text = inputs['text']
+    language = inputs['language']
+    text = ' '.join(text.split())
+    for p in ',.:;?!-_':
+        text = text.replace(p, '')
+    text = text.lower().split()
```

Comment:
No need to process text in any way.

Reply:
snippet copied from preprocess_all). Should I remove it there as well?